nf_core/lint/files_unchanged.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python
+
+import logging
+import filecmp
+import os
+import tempfile
+
+import nf_core.create
+
+
+def files_unchanged(self):
+    """Checks that certain pipeline files are not modified from template output.
+
+    Iterates through the pipeline's directory content and compares specified files
+    against output from the template using the pipeline's metadata. File content
+    should not be modified / missing.
+
+    Files that must be unchanged::
+
+        'CODE_OF_CONDUCT.md',
+        ['LICENSE', 'LICENSE.md', 'LICENCE', 'LICENCE.md'], # NB: British / American spelling
+        '.gitattributes',
+        '.github/.dockstore.yml',
+        '.github/CONTRIBUTING.md',
+        '.github/PULL_REQUEST_TEMPLATE.md',
+        '.github/markdownlint.yml',
+        '.github/ISSUE_TEMPLATE/bug_report.md',
+        '.github/ISSUE_TEMPLATE/config.yml',
+        '.github/ISSUE_TEMPLATE/feature_request.md',
+        '.github/workflows/awsfulltest.yml',
+        '.github/workflows/awstest.yml',
```

Comment:
`TODO` string should be removed from the template then, if it can't be edited

Reply:
File removed from test 👍🏻